tests/test_2fa_duo.py

```diff
@@ -169,11 +173,11 @@ def test_2fa_session_multiple_crn(self, netid10, enter_duo_passcode):
             self.browser.find_element_by_xpath("//input[@value='sptest07']").click()
             self.browser.snap()
             self.browser.click(Locators.submit_button)
-            self.browser.wait_for_tag('p', "Using your 'sptest07' Duo identity.")
+            sleep(5)
```

Comment:
same ask as above (below?)

1. try using wait here if you haven't yet
2. add a comment indicating what's being waited for with the sleep

(sorry if this is getting repetitive - if you'd prefer "can you do this for all `sleep()`s" let me know)

Reply:
removed